locale/en_US/manager.po

```diff
@@ -2141,3 +2141,27 @@ msgstr "The full names of the authors"
 
 msgid "emailTemplate.variable.submission.submissionUrl"
 msgstr "The URL to the submission in the editorial backend"
+
+msgid "mailable.mailDiscussionMessage.name"
+msgstr "Discussions"
+
+msgid "mailable.mailDiscussionMessage.description"
+msgstr "Automatic email sent to participants of a new or updated discussion"
+
+msgid "mailable.mailReviewerAssigned.name"
+msgstr "Reviewer Assigned"
+
+msgid "mailable.mailReviewerAssigned.description"
+msgstr "Email templates available when a reviewer is assigned"
+
+msgid "mailable.mailReviewerReinstate.name"
+msgstr "Reviewer Reinstate"
+
+msgid "mailable.mailReviewerReinstate.description"
+msgstr "Email templates available when a reviewer that was unassigned is reinstated"
```

Comment:
This email is sent when a reviewer that was unassigned is reinstated by an editor.